applications/StructuralMechanicsApplication/tests/test_patch_test_small_strain.py

```diff
@@ -190,7 +185,7 @@ def _check_outputs(self,mp,A,dim):
             for strain in out:
                 for i in range(len(reference_strain)):
                     if abs(strain[i]) > 0.0:
-                        self.assertLess((reference_strain[i] - strain[i])/strain[i], 1.0e-6)
+                        self.assertLess((reference_strain[i] - strain[i])/strain[i], self.tolerance)
 
```

Comment:
I can't add a suggestion because github thinks there's a deleted line in the middle, but relative errors should be computed relative to the reference value (+ missing checks on the ```else``` branch):

```py
                    if abs(reference_strain[i]) > 0.0:
                        self.assertLess(abs((reference_strain[i] - strain[i])/reference_strain[i]), self.tolerance)
                    else:
                        self.assertLess(abs(strain[i]), self.tolerance)
```